lib/password_template.html

```diff
@@ -77,14 +78,7 @@
         .staticrypt-content {
             height: 100%;
             margin-bottom: 1em;
-            background: #76b852; /* fallback for old browsers */
-            background: -webkit-linear-gradient(right, #76b852, #8DC26F);
-            background: -moz-linear-gradient(right, #76b852, #8DC26F);
-            background: -o-linear-gradient(right, #76b852, #8DC26F);
-            background: linear-gradient(to left, #76b852, #8DC26F);
-            font-family: "Arial", sans-serif;
-            -webkit-font-smoothing: antialiased;
-            -moz-osx-font-smoothing: grayscale;
```

Comment:
I'm fine with removing the gradient but the CSS related to font should stay

Reply:
Oops. Sorry!